tests/testutil.hpp

```diff
@@ -37,6 +37,7 @@
 #endif
 #include "../include/zmq.h"
 #include "../src/stdint.hpp"
+#include "../src/fd.hpp"
```

Comment:
Please don't - I do not want to start including headers from the tests. Yes there's already one or two - they should be removed, not the other way around.

Reply:
Fixed